indico/modules/rb_new/client/js/common/filters/validation.js

```diff
@@ -0,0 +1,45 @@
+/* This file is part of Indico.
+ * Copyright (C) 2002 - 2018 European Organization for Nuclear Research (CERN).
+ *
+ * Indico is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License as
+ * published by the Free Software Foundation; either version 3 of the
+ * License, or (at your option) any later version.
+ *
+ * Indico is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Indico; if not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {toMoment} from 'indico/utils/date';
+import {Translate} from 'indico/react/i18n';
+import {handleAxiosError} from 'indico/utils/axios';
+import {resetPageState} from '../../actions';
+
+
+const mixedValidator = (filters) => {
+    let valid = true;
+    if (!('timeSlot' in filters)) {
+        return valid;
+    }
+
+    const {timeSlot: {startTime: st, endTime: et}} = filters;
+    if (st && et && toMoment(st, 'HH:mm').isSameOrAfter(toMoment(et, 'HH:mm'))) {
+        valid = false;
+    }
+    return valid;
+};
+
+export const validFilters = (filters, namespace, dispatch) => {
+    if (!filters.error && mixedValidator(filters)) {
+        return true;
+    }
+    const errorMsg = Translate.string('The provided URL was not properly formatted, please try again.');
+    handleAxiosError({message: errorMsg});
```

Comment:
I don't think this is the right thing to use here. It's meant for errors from requests.